server/controllers/CommentController.js

```diff
@@ -240,6 +240,43 @@ class CommentController {
       comment,
     });
   }
+
+  /**
+  * @description This method deletes a comment on an article
+  * @param {object} req - exporess request object
+  * @param {object} res - express response object
+  * @param {object} next - express next object
+  * @returns {object} returns an object of comment
+  */
+  static async deleteComment(req, res, next) {
+    const userId = req.userData.id;
+    const { commentId } = req.params;
+
+    let comment;
+    try {
+      comment = await Comment.findByPk(commentId, {
+        raw: true,
+      });
+
+      if (comment.userId !== userId) {
```

Comment:
This logic is not really necessary. The validateAccess middleware with the help of the validateResourceId middleware has taken care of this.

Reply:
This is fixed now @augustineezinwa.